projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -226,32 +269,24 @@ public function empty_result() {
 	 * @return array
 	 */
 	public function get_site_resource_from_wpcom( $req ) {
-		// so currently it returns 403 for posts.
 		$resource = $req->get_param( 'resource' );
 		switch ( $resource ) {
 			case 'site-has-never-published-post':
 				return $this->get_has_never_published_post( $req );
 
+			// TODO: remove the following calls from the frontend.
```

Comment:
These are placeholders too, otherwise the frontend doesn't function normally, which are due to be removed.

Reply:
Maybe it would help to first remove them from the frontend, then resume working on this PR so it's very clear which endpoints we need?